pa11ycrawler/spiders/edx.py

```diff
@@ -142,7 +142,7 @@ def __init__(
                     all_blocks="true",
                 )
             )
-            self.start_urls = [api_url]
+            self.start_url = api_url
```

Comment:
In general I'm trying to understand why we don't use a list. I thought that was something that was useful for Scrapy...